helm/k8s/templates/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.container.name }}
+  namespace: {{ .Values.environment }}-{{ .Values.container.name }}
+spec:
+  replicas: {{ .Values.replicas }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.container.name }}
+        environment: {{ .Values.environment }}
+    spec:
+      containers:
+        - name: {{ .Values.container.name }}
+          image: {{ .Values.container.repo }}/{{ .Values.container.name }}:{{ .Values.container.tag }}
+          ports:
+            - containerPort: {{ .Values.container.port }}
+          env:
+            - name: GREETING
```

Comment:
Let's make it a `volumeMount` instead—I cling to my belief that environment variables are a deprecated legacy thing, so we ought to set a more modern example. (In Docker, environment variables are all you have, of course, so you can see why some people still like to use them.)